### ISSUE TITLE
New ResearchHub Homepage

### ISSUE THREAD

**Issue by yattias** [ResearchHub/issues#39]
The goal of this issue is to solicit ideas from the team and the community about what the ResearchHub homepage could look like. More specifically:

- What type of content would you like to see in it?
- How granular should the content be (e.g. Peer Review or comment vs. paper) ?
- What kind of social interactions should be included? (upvotes, following activity, etc...)
- Is there anything in particular you think we should eliminate from our current homepage?

Thank you for taking the time to provide feedback

**Comment by dominikusbrian** [comment]
Excited to learn that there will be upgrade for ResearchHub Homepage. While the discussion is just starting, I would love to contribute some ideas that can be useful and meaningful addition to what already exist / planned.

1. **[Latest in Science]** A section, feed stream, or special tab that contains the latest on science. Doesn't need to be exhaustive but possibly contain a list of one's favorite journal, or big breakthrough on the news and the like.  As an inspiration for this feature the https://www.researcher-app.com/ interface might have an item or two ResearchHub can consider. The main interface for their feed can either be journal frontpage or simply name. Then when clicked there will be feed on various latest publication on the journal. Just like researchhub for open-access or freely available content full paper/ article can be shown. While for most case it is mainly for browsing through latest literature and marking things for read or saved them to proper categories. RH already have much of these functionalities on the backend side so having something like this on the frontend would be nice.
![image](https://github.com/ResearchHub/issues/assets/60097047/08894964-d366-4aaa-9ff3-e3c3a636ab41) I been an active user for Researcher-app for several years (back when I still need to commute) and felt they help me a lot in getting up-to-date with latest in science on my way to work. So I don't need to be drowning in journal subscription emails to keep up.

2. **[Content type based granularity]** Granularity wise, for comment and paper or peer review (basically the human contributed content) having a short paragraphs of abstract or the contents are desirable. However, with the peer review bounties currently there are too much redundancies. Ideally only the header and title of paper will suffice, while the rest of template based guideline for peer review can remain hidden/ not-shown (at least on homepage appearance).

3. **[Share to Social -- External Social]** A button to share on socials can go a long way in driving traffics to ResearchHub and make it convenient for user to share their thought/content they find useful. Modelling the share button like the one in Medium will be one option. 
![image](https://github.com/ResearchHub/issues/assets/60097047/b1655516-c984-454d-85b3-e4b63fd78d51)

4. **[RH Read Counts (for post/paper) -- Internal Social]** Having a read count displayed (and maybe  accounted for deciding what's trending) would be beneficial stats  to track and learn about. 

5. **[Wild Card Ideas -- Internal Social]** 
a. ResearchHub in-platform citation ? a feature to quickcite other post/paper natively in ResearchHub as an addition to the tools here under the type box for posting comment. 
![image](https://github.com/ResearchHub/issues/assets/60097047/9d19b364-67e3-4933-9503-d27f7055f764) The user can then simply click the [cite button] which will pop out a modal similar to what is shown in search bar, user can then select the paper/post which then will automatically show the in-text display citation. Either in number format "[1]", or maybe following the usual APA, Chicago or other standard preferred. Perhaps better to set it to one format for ResearchHub to keep things simple and clean. 

b. Make the [Explore-For You] tab section more personalized. For instance new user will have nothing here, and the content suggested will simply be hubs/authors/post "followed/subscribed" by the user. This makes  things clean for user who want to get direct feed streams of things they mainly concerned or have interest about. When user want to simple get overall feed, they can go to [Explore- Frontpage] or [Live Feed] as usual.

6. **[Things to consider for elimination]**
The redundancies of bounty guideline text:
![image](https://github.com/ResearchHub/issues/assets/60097047/12199354-913f-43c5-aeb3-a4fb0bb91698)
all else are actually already pretty sleek and clean. Great job by the dev team.

**Comment by Franklynstein** [comment]
Hello @yattias , Jeffery Koury informed me to go through the issues and apply for the ones I could do so they get assigned to me by the lead devs. Would paste this comments under the ones I believe I can work on, kindly assign them so I can begin work on them.

**Comment by yattias** [comment]
@Franklynstein Are you a designer? If so, you can submit some design ideas and/or mocks in here

**Comment by Dolosus19** [comment]
What type of content would you like to see in it?
-Could the bounties live feed look more like the home page feed? Or perhaps have a preview of the comment nested under the paper title?
-Can we have a page browsing function like Google?
-Perhaps there could be a permanent rotating canner that advertises the newest/latest big new items from ResearchHub. This would be located on top of the "Frontpage" and "For You" tabs.
-I think that the bar on the tight should not scroll away. I think that it would better if it was locked in place. The recently peer reviewed and open ballots are a little cluttered. Perhaps it should just be a box that people can click on (similar to Reddit)
-Having the same options in "For You" page as the Frontpage 
![image](https://github.com/ResearchHub/issues/assets/113945454/53b0b8a8-95ee-48ee-9984-2f2c0af85b10)
-I would also like to see a dark-mode introduced.


How granular should the content be (e.g. Peer Review or comment vs. paper) ?
-On the homepage, I think that the papers should be highlighted. However, the ability to quickly see a preview each type of comment would be nice to have (this would be like where the paper icon is, there are tabs for comments, bounties, and peer reviews, and by clicking on them it would drop down and show a preview of what was written).

What kind of social interactions should be included? (upvotes, following activity, etc...)
-I would like to see the ability to follow other users.
-I would like to be able to share to other social media sites.
-Being able to specifically follow the updates on a particular paper would be helpful as well.

Is there anything in particular you think we should eliminate from our current homepage?

**Comment by arshiamalek** [comment]
Hi @yattias, 

Here are some of my thoughts:

**General**
- Dark Mode
- Change "Lab Notebook" to "Notebook" (allow for more inclusivity for other disciplines that do not have traditional labs)
- Perhaps a bit more color to _modernize_ the appearance (RH logo color scheme)
- Ensuring links are accurate (i.e. _Jobs_ link is linked to an outdated Notion page from 2021)
- Add _icon_ for LinkedIn (we need more people directed to follow our LinkedIn page)
- Keep the "Live" page! One of my personal favorite features to see what's going on _live_

**Peer Review Section on Homepage**
- Create a new tab for peer reviews
- Users can discover peer review options and available bounties 
- Comprehensive filtering option (field, date, availability, contact info of associated editor)
- Tracking tab/page should highlight the preprint in question and perhaps some key characteristics about it vs. the current view which highlights the body of the assigned bounty. It's a bit backwards. (Please see image below) 

<img width="852" alt="Screenshot 2024-06-17 at 3 41 11 PM" src="https://github.com/ResearchHub/issues/assets/173108476/7517b3b2-e4f4-4660-afe7-47a31481909e">

As always, thank you and the rest of the engineering team for all that you do for RH.

Arshia

**Comment by Franklynstein** [comment]
> @Franklynstein Are you a designer? If so, you can submit some design ideas and/or mocks in here

I am a developer not a designer tho, I assumed the design had been finalized

**Comment by Franklynstein** [comment]
> What type of content would you like to see in it? -Could the bounties live feed look more like the home page feed? Or perhaps have a preview of the comment nested under the paper title? -Can we have a page browsing function like Google? -Perhaps there could be a permanent rotating canner that advertises the newest/latest big new items from ResearchHub. This would be located on top of the "Frontpage" and "For You" tabs. -I think that the bar on the tight should not scroll away. I think that it would better if it was locked in place. The recently peer reviewed and open ballots are a little cluttered. Perhaps it should just be a box that people can click on (similar to Reddit) -Having the same options in "For You" page as the Frontpage ![image](https://private-user-images.githubusercontent.com/113945454/340154769-53b0b8a8-95ee-48ee-9984-2f2c0af85b10.png?jwt=eyJhbGciOiJIUzI1NiIsInR5cCI6IkpXVCJ9.eyJpc3MiOiJnaXRodWIuY29tIiwiYXVkIjoicmF3LmdpdGh1YnVzZXJjb250ZW50LmNvbSIsImtleSI6ImtleTUiLCJleHAiOjE3MTg3ODY5MzAsIm5iZiI6MTcxODc4NjYzMCwicGF0aCI6Ii8xMTM5NDU0NTQvMzQwMTU0NzY5LTUzYjBiOGE4LTk1ZWUtNDhlZS05OTg0LTJmMmMwYWY4NWIxMC5wbmc_WC1BbXotQWxnb3JpdGhtPUFXUzQtSE1BQy1TSEEyNTYmWC1BbXotQ3JlZGVudGlhbD1BS0lBVkNPRFlMU0E1M1BRSzRaQSUyRjIwMjQwNjE5JTJGdXMtZWFzdC0xJTJGczMlMkZhd3M0X3JlcXVlc3QmWC1BbXotRGF0ZT0yMDI0MDYxOVQwODQzNTBaJlgtQW16LUV4cGlyZXM9MzAwJlgtQW16LVNpZ25hdHVyZT1jMzAzZWU2MmM2ZDdjYTljN2E2YTYzM2Y5YjM4NDY5MjkzODQyMTM5NzYwOGNkZGQ0YzFlOTkxZWM4ZjEzY2QwJlgtQW16LVNpZ25lZEhlYWRlcnM9aG9zdCZhY3Rvcl9pZD0wJmtleV9pZD0wJnJlcG9faWQ9MCJ9.DjpdgGNEHCtNHV-WQpeZsMPccaEU1ZrBce3KCsOkfGU) -I would also like to see a dark-mode introduced.
> 
> How granular should the content be (e.g. Peer Review or comment vs. paper) ? -On the homepage, I think that the papers should be highlighted. However, the ability to quickly see a preview each type of comment would be nice to have (this would be like where the paper icon is, there are tabs for comments, bounties, and peer reviews, and by clicking on them it would drop down and show a preview of what was written).
> 
> What kind of social interactions should be included? (upvotes, following activity, etc...) -I would like to see the ability to follow other users. -I would like to be able to share to other social media sites. -Being able to specifically follow the updates on a particular paper would be helpful as well.
> 
> Is there anything in particular you think we should eliminate from our current homepage?

I can implement this additions

**Comment by barmstrong** [comment]
A couple thoughts on the homepage

1. personalization

I don't feel like the home feed is personalized to my scientific interests today so I'm less likely to find something to engage with, or come back. I like the idea of having a "following" feed based on the hubs I follow, but even more important would be an algorithmic feed based on many factors (hubs i follow, posts i've interacted with in the past, posts people in general are interacting with so I can discover new areas of interest, etc - there could be unlimited number of variables that go into a ML ranking algorithm). These algorithmic feeds have proven successful across most sites with network effects, as they don't require as much work from the user to curate their following list (hence X, TikTok, Insta, LinkedIn, Spotify, YouTube, NetFlix etc have all moved to algorithmic feeds). We'd need to ensure it doesn't devolve into pop science and some level of quality is factored in - and I think this could be possible by weighting votes by user reputation, but my guess is an algorithmic feed would create the most value if well executed (tuning the ML right is non-trivial). Think about the quality of recommendations on YouTube as an example - it manages to be quite high (by YouTube standards - best content they have). It's ok if our content doesn't look/feel exactly like a stuffy/formal journal - the test for me is whether it's relevant to the individual looking at it, and useful to accelerating science. X uses a metric like "non-regretted user minutes" but it's not clear to me how they measure this (user surveys?)

2. density

On the homepage today I can only see about 4 items above the fold. This is not crazy - for instance Reddit and X show about this many in standard format. Other sites like Hacker News show way more though (23 items). It could be worth exploring how many items we show with greater density. The larger issue is not really density though (this is just one variable to play with) it is....

3. engagement

Related to the first two points - to me it's really about how we drive more engaging content on the homepage, so that, on average, when you load up the homepage you see something that is interesting, relevant to your work, and helpful. Something that accelerates science...for you. 

The mental model i have of any user is they are busy, skimming (not reading), and will typically hit a back button with 2-5 seconds if they don't think they are in the right spot. So our biggest competitor is the back button. How do we capture someones attention on the first/second/third visit where they see something interesting enough to pause and take a next step (whatever that is - such as signing up). Many users will start by being read only at first (95%+) and only after reading many times will they sometimes sign up and start to contribute (9%) in some small way like commenting and upvoting. Only the remaining 1% will ever graduate to contributing meaningful top quality content (original posts, peer reviews, funding research, etc).

So it's a funnel - and the top of our funnel is a leaky bucket right now. What are the sorts of thing that might drive engagement?
- personalized content we already talked about
- more images? videos? 
- user profile photos (these are currently missing from the page)
- credibility indicators (view/read counts, follower/rep counts for individuals, etc)
- money/value (RSC earned, or being raised)
- drama (hate to say it but this drives a lot of engagement, and we don't want too much of this - but I could see there being versions of this which are appropriate, such as calling out bad science that doesn't reproduce, or with falsified data, which would help with the replication crisis, or people hotly debating whether something is true or not based on the evidence, this would help advance science)

4. literature reviews

Slightly adjacent to the homepage topic specifically, but literature reviews are a post type that we've explored/debated before, which could be additive on the homepage. A major topic right now in science and society more broadly is how we should know if something is true. Should we trust "experts"? Who is an expert after all? There have been various examples (lab leak theory, the food pyramid etc) where experts have said one thing, and it turned out something else could be true. If we could make a meta layer on top of research papers/posts, which is to collect those papers into a crowd sourced meta-study or literature review, which tries to make an assessment of the veracity of a statement, based on the best available evidence, this could be a valuable unit of work on the internet that doesn't quite exist (although arguably ChatGPT is approximating this).

5. entirely different content

It’s possible optimizing the current feed (of papers, posts, etc) is the wrong place to focus - and we should be thinking about a valuable feature (like funding, bounties, etc) and make the site primarily about this vs the feed. It's possible that curating which papers to see is not solving a real problem for users, and something else we're building should be the focus on the homepage.

**Comment by jeffreykoury** [comment]
There's already quite a bit of feedback, so I'll just double down on what I agree with as well as some extra points

**Double down points**
1. Defaulting to the "For you" feed seems good, and incorporating some input parameters to surface content to the top like altmetric type of inputs (i.e. papers with a ton of conversation on them on X are generally impactful and/or controversial and either way deserve some eyes on them to allow for public discourse). Also adding filters in the "For You" section like @Dolosus19 mentioned would be necessary too. 
2. At least for peer review bounties, changing the cards of peer review bounties to give more room for the title of the paper rather than the body of the bounty message. From the prospective peer review perspective, more context can be gleened from the title.
3. Agree with @arshiamalek for having a dedicated peer review tab/section.

**Opposite Perspective**
1. I know some people like a share to socials button, and @dominikusbrian mentioned it above, but I'm personally not a fan because it generates a cookie cutter post and feels less personable. IMO people will share the content in an organic way to other socials if they're just excited and passionate about their experience
2. Granularity in the feed: Although from a visually appealing perspective I think commentary with images/videos would add a visual aesthetic-ness to the livefeed, I worry that going into this granularity would not give people enough context as they're scrolling. **What might be the best of both worlds is keep the high level paper card in the feed, but also have a connecting line (like threads within an X post) showing more granular conversations/peer reviews from within the paper right underneath the paper card.** This way you keep the context from the paper card, while adding the granularity and visual appeal of images/videos from conversation.
 
 **Extra points**
1. Difficulty with discoveribility of open peer review bounties has been some feedback we've received, and so a whole section for peer reviews would be great but maybe for now making the "open peer review bounties" on the right side show up above "peer reviews completed" so it's visible without scrolling. Related to this is we've gotten feedback for needing improved keyword searching to help discover these open bounties easier. 
2. For our system of peer reviews the way I think about these is essentially _A Marketplace for Peer Review_. So as a user is scrolling through the For You/livefeed it'd be interesting to think about an easy button to "request to peer review" in the paper card if the paper is a preprint. A user can quite easily discover or upload preprints on RH that overlap with their expertise, but they're limited to open peer review bounties we assign, so a request to peer review would make their reviewing experience more seamlessly 
4. A "Press Kit" link at the bottom of the left bar linking to a google drive which contains RH's high res logos

**Comment by arshiamalek** [comment]
I agree with you @jeffreykoury allowing users to **_request to peer review_** preprints. This will make it easy for us administrators to track, approve, and assign a bounty quickly.
